vcap_utils/vcap_utils/backends/base_openvino.py

```diff
@@ -35,34 +35,112 @@ def __init__(self, model_xml: bytes,
 
         self.ie = ie_core or IECore()
 
-        # Find the optimal number of InferRequests for this device
-        supported_metrics = self.ie.get_metric(
-            device_name, _SUPPORTED_METRICS)
-        if _RANGE_FOR_ASYNC_INFER_REQUESTS in supported_metrics:
-            low, high, _ = self.ie.get_metric(
-                device_name, _RANGE_FOR_ASYNC_INFER_REQUESTS)
-            # Cap the n_requests, because sometimes high_n crashes the system
-            # TODO(Alex): Figure out _why_ hddl crashes when set to 'high'
-            n_requests = max(0, min(low * 2, high))
-        else:
-            # Use the devices default
-            n_requests = 0
+        # If the device is a MULTI device, use the default num_requests
+        is_multi_device = device_name.lower().startswith("multi")
```

Comment:
The purpose of this change is to allow "MULTI:*" devices to have the default num_requests, but for non "MULTI" devices we query the devices metrics to find out the best num_requests